### PR TITLE
Replaces the RD suit with the half life HEV suit

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

gives the RD hardsuit an sprite that looks like an HEV suit, changes the name and desc

## Why It's Good For The Game

cause muh refferences
## Changelog
:cl:
add: Added hev suit from half life
/:cl:

### CODE REVIEW

**Comment by qwertyquerty** [comment]
but why, also you have grammar issues

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 1]
```diff
@@ -449,8 +449,8 @@
 
 	//Research Director hardsuit
 /obj/item/clothing/head/helmet/space/hardsuit/rd
-	name = "prototype hardsuit helmet"
-	desc = "A prototype helmet designed for research in a hazardous, low pressure environment. Scientific data flashes across the visor."
+	name = "Hazardous Envoirment Suit Helmet"
+	desc = "An Hardsuit designed after an Old-Earth type exosuit renewed by Nanotrasen"
```
Comment (qwertyquerty) [inline code/modules/clothing/spacesuits/hardsuit.dm]: "an hardsuit"

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 2]
```diff
@@ -478,8 +478,8 @@
 
 /obj/item/clothing/suit/space/hardsuit/rd
 	icon_state = "hardsuit-rd"
-	name = "prototype hardsuit"
-	desc = "A prototype suit that protects against hazardous, low pressure environments. Fitted with extensive plating for handling explosives and dangerous research materials."
+	name = "hazardous envoirment suit"
+	desc = "An hardsuit designed after an Old-earth type exosuit, renewed by Nanotrasen."
```
Comment (qwertyquerty) [inline code/modules/clothing/spacesuits/hardsuit.dm]: "an hardsuit"

Comment (ghost) [reply code/modules/clothing/spacesuits/hardsuit.dm]: why not, and also english is not my primary language

Comment (qwertyquerty) [reply code/modules/clothing/spacesuits/hardsuit.dm]: should be "a hardsuit"

Comment (ghost) [reply code/modules/clothing/spacesuits/hardsuit.dm]: fixed.

**Comment by qwertyquerty** [comment]
Did you test this?

**Comment by qwertyquerty** [comment]
Show me an image of the suit

**Comment by ike709** [comment]
@qwertyquerty 
image

**Pull request merged**